db_operations.py

```diff
@@ -0,0 +1,74 @@
+import models
```

Comment:
In this file, the method `one()` should be used, when expecting a single result, instead of `first()` or `all()` as the latter methods imply you are expecting multiple results.

Also when using `one()` you should handle the possible exceptions accordingly.